### PR TITLE
Fix crash on processing devtools event

### PR DESCRIPTION
**Pull request by annagrin**
Fix crash on processing devtools event.

Closes: https://github.com/dart-lang/webdev/issues/1874

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 analyzer:
-  strong-mode:
-    implicit-casts: false
```
Comment (annagrin) [inline analysis_options.yaml:3]: Newest analyzer gives a warning about implicit-casts not being supported.

Comment (elliette) [reply analysis_options.yaml:3]: Just added with https://github.com/dart-lang/webdev/pull/1863 :)

Comment (annagrin) [reply analysis_options.yaml:3]: with sync:)

Comment (annagrin) [reply analysis_options.yaml]: done!

File: dwds/lib/src/dwds_vm_client.dart [hunk 2]
```diff
@@ -152,18 +152,22 @@ void _processSendEvent(Map<String, dynamic> event,
         _logger.finest('Received DevTools event: $event');
         final action = payload?['action'] as String?;
         final screen = payload?['screen'] as String?;
+        final devToolsStart = dwdsStats.devToolsStart;
+        final debuggerStart = dwdsStats.debuggerStart;
         if (screen != null && action == 'pageReady') {
```
Comment (elliette) [inline dwds/lib/src/dwds_vm_client.dart:157]: Could we pull everything in this `if` statement into a method (eg, `_recordDevToolsReadyTimes`

Comment (annagrin) [reply dwds/lib/src/dwds_vm_client.dart:155]: Done!

**Review by elliette — APPROVED** [review]

**Pull request merged**